package.json

```diff
@@ -11,7 +11,7 @@
         "@headlessui/vue": "^1.7.15",
         "@heroicons/vue": "^2.0.18",
         "@inertiajs/inertia-vue3": "^0.6.0",
-        "@inertiajs/progress": "^0.2.7",
+        "@inertiajs/progress": "^0.1.2",
```

Comment:
Why we downgrade version of this package?

Reply:
fixed